### DISCUSSION TITLE
Unique idea, Option to delete played Beatmap after completing

### DISCUSSION THREAD

**Discussion by DBeitniz** [ppy/osu#27291] [Ideas]
So I have a little suggestion, that could be a cool thing, if you would add a button to chose if you want to delete the Beatmap you played, because I often play Multiplayer and I often just get some sh*tty maps and to delete it I need to go there and there... Just to much, add a button to on ranking screen in multiplayer to delete map you played right now.

**Comment by nagi-desuuu** [comment]
I mean, there's the Last Played sorting in song select... After a session or two, maybe try going there and remove them like normal?

**Reply by DBeitniz** [reply]
Yeah, but if you like a song and all the other were trash, and you already don't know which song that was (Happens to me when I play for a hour or so), so it isn't that great, or if you are in cool lobby, you just don't wanna leave it, after you found a good song or mapping.
